README.md

```diff
@@ -41,59 +41,83 @@ These [values are what we value](http://www.slideshare.net/reed2001/culture-1798
 The initial list of these values were created at our first company retreat, when Bonusly was a team of seven people.
```

Comment:
Add in a sentence about the adjustment process for this new version?

Reply:
+1